tests/bin/test.docker-helper.bats

```diff
@@ -99,48 +120,46 @@ setup_file() {
 
 @test "push pushes built image wo versions" {
   export IMAGE_NAME="localstack/test"
-  export MAIN_BRANCH="`git branch --show-current`"
+  export MAIN_BRANCH="main"
   export DOCKER_USERNAME=test
   export DOCKER_PASSWORD=test
   export PLATFORM=arm64
-  export FORCE_VERSION_TAG_PUSH=0
-  export VERSION_FILE="$BATS_TEST_DIRNAME/files/VERSION"
+  export TEST_SPECIFIC_VERSION="3.6.1.dev45+g15cb62bd5.d20240814"
```

Comment:
Comment by silvio during our call: We can remove the local part here, as it is not present anyway.